File: asp.net/sdk-lpl-mvc5/sdk-lpl-mvc5/Models/InformationCompteModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace sdk_lpl_mvc5.Models
{
    /// <summary>
    /// Modèle pour la création d'un lien d'inscription partenaire
    /// Représente le modèle de données qui compose le paramètre <user> dans l'url d'inscription partenaire
    /// </summary>
    public class InformationCompteModel
    {
        public string Email { get; set; }
        public string Pseudo { get; set; }
    }
}

File: nodejs/models/InformationCompteModel.js
/**
* Modèle pour la création d'un lien d'inscription partenaire
* Représente le modèle de données qui compose le paramètre <user> dans l'url d'inscription partenaire
* @author  Bastien Caubet <bastien@nextinpact.com>, Luc Raymond <luc@nextinpact.com>
*/
module.exports = InformationCompteModel;

function InformationCompteModel(mail, userName) {
    this.Email = mail;
    this.Pseudo = userName;
}